src/composables/transaction.ts

```diff
@@ -105,12 +129,7 @@ export const useTransactionToken = (
     isAllowance,
     isErrorTransaction,
     isDex,
-    availableTokens,
-    getTxAmountTotal,
-    getTxSymbol,
-    getTxType,
-    getTxDirection,
-    getDexContracts,
+    transactionOwnershipStatus,
```

Comment:
I suggest to align the returned property names with one naming pattern.
```
transactionSomething / isTransactionSomething
```
or
```
something / isSomething
```
Now its mixed.